problemsets-mat223/modules/module10-exercises.tex

```diff
@@ -1,10 +1,52 @@
 \begin{exercises}
+    %45 min
+    %60 min
 	\begin{problist}
-		\prob  Example problem with parts
+	    \prob Let $\mathcal R:\R^2\to\R^2$ be rotation clockwise by $135^\circ$, given by some matrix $M_R$. Compute $M_R$.
+	    
+	    \prob Let $\mathcal P:\R^2\to\R^2$ be projection onto the $y$-axis, given by some matrix $M_P$. Compute $M_P$.
+	    
+	    \prob Let $\mathcal F:\R^2\to\R^2$ be a transformation flipping over the line $y=x$, given by some matrix $M_F$. Compute $M_F$.
+	    
+	    \prob Let $\mathcal S:\R^2\to\R^2$ be a transformation stretching by 2, given by some matrix $M_F$. Compute $M_F$.
+	
+		\prob Let $\mathcal U:\R^2\to\R^2$ be the transformation given by $M=\matc{0&0\\-\sqrt{2}/2 & -\sqrt{2}/2}$, let $\mathcal R$ and $\mathcal P$ be the transformations given above.
 		\begin{enumerate}
-			\item   The first part
-			\item   The second part
+			\item   compute the matrix $M_1$ that gives $\mathcal R \circ \mathcal P$.
+			\item   compute the matrix $M_2$ that gives $\mathcal P \circ \mathcal R$.
+			\item   Write $\mathcal U$ as the composition (in some order) of $\mathcal R$ and $\mathcal P$.
 		\end{enumerate}
+		
+		\prob Let $\mathcal V:\R^2\to\R^2$ be the transformation given by $M=\matc{0&2\\2&0}$, let $\mathcal F$ and $\mathcal S$ be the transformations given above.
+		\begin{enumerate}
+			\item   compute the matrix $M_1$ that gives $\mathcal F \circ \mathcal S$.
+			\item   compute the matrix $M_2$ that gives $\mathcal S \circ \mathcal F$.
+			\item   Write $\mathcal U$ as the composition (in some order) of $\mathcal F$ and $\mathcal S$.
+		\end{enumerate}
+		    
+		\prob Let $\mathcal A: \R^2 \to \R^2$ and $\mathcal B: \R^2 \to \R^2$ be matrix transformations with matrices
+		\[
+		    M_A = \matc{2&2\\1&3} \quad and \quad M_B = \matc{3&2\\0&4}
+		\]
+		Define $\mathcal T = \mathcal A \cdot \mathcal B$, we know $\mathcal T$ has a matrix $M_T$. 
+		\begin{enumerate}
+		    \item Find $M_T$ by computing some input-output pairs for $\mathcal T$.
+		    \item Find $M_T$ by applying the theorem we learned in this module. 
```

Comment:
-> "by using matrix multiplication applied to $M_A$ and $M_B$"

Reply:
That is strange, I did change in my file here...